gears/src/main/scala/nl/lumc/sasc/biopet/pipelines/gears/GearsReport.scala

```diff
@@ -41,120 +52,493 @@ object GearsReport extends MultisampleReportBuilder {
 
   def reportName = "Gears Report"
 
-  override def extFiles: List[gears.GearsReport.ExtFile] = super.extFiles ++ List("js/gears.js", "js/krona-2.0.js", "img/krona/loading.gif", "img/krona/hidden.png", "img/krona/favicon.ico")
-    .map(x => ExtFile("/nl/lumc/sasc/biopet/pipelines/gears/report/ext/" + x, x))
+  override def extFiles: List[gears.GearsReport.ExtFile] =
+    super.extFiles ++ List("js/gears.js",
```

Comment:
I think gears.js can be removed from this. This is not used anymore since the implementation of krona